docs/literate/src/files/first_steps/create_first_setup.jl

```diff
@@ -113,8 +116,9 @@ end
 
 semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver;
                                     source_terms = source_term_exp_sinpi)
+#
```

Comment:
```suggestion
# which leaves us with an ODE problem in time with a span from 0.0 to 1.0.
# This approach is commonly referred to as the method of lines. 
```
I would move the two lines 114/115 from above here between the two code blocks for better readability. Obviously, they have to be deleted above...